### PR TITLE
[ADD] hr_payslip_run_print_slips

### PR DESCRIPTION
**Pull request by saltonmassally**
Print all slips in payslip run

### CODE REVIEW

File: hr_payslip_run_print_slips/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Configuration
+=============
+
+This module does not require any special configuration apart from enabling it as
+shown above.
+
+Usage
+=====
+
+Print slips from payslip run
```
Comment (feketemihai) [inline hr_payslip_run_print_slips/README.rst:30]: Please add the Try me on Runbot section...

After that :+1:

**Comment by saltonmassally** [comment]
@feketemihai button added

**Review by feketemihai — APPROVED** [review]
Not tested, code looks fine.

**Comment by feketemihai** [comment]
Please reopen the PR if it's important to you.

**Pull request closed without merging**